"config.h" not found (CocoaPods+iOS7)

Hi,

I'm working on iOS 7, using Xcode 5 (preview of course), and when I compile my project, I get this error:

```
Mantle/metamacros.h:13:10: fatal error: 'config.h' file not found
```

Using `:head` for the pod.

Any idea?

Thanks,
Jérémy

--- a/Mantle/extobjc/metamacros.h
+++ b/Mantle/extobjc/metamacros.h
@@ -9,10 +9,6 @@
 #ifndef EXTC_METAMACROS_H
 #define EXTC_METAMACROS_H
 
-#ifdef HAVE_CONFIG_H
-#include "config.h"
-#endif
-
 /**
  * Executes one or more expressions (which may have a void type, such as a call
  * to a function that returns no value) and always returns true.

